hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/BucketManager.java

```diff
@@ -101,4 +103,43 @@ void applyDeleteBucket(String volumeName, String bucketName)
   List<OmBucketInfo> listBuckets(String volumeName,
       String startBucket, String bucketPrefix, int maxNumOfBuckets)
       throws IOException;
+
```

Comment:
Can we abstract these 4 acl operations into a separate ACL interface so that we can reuse it for volume/bucket/key/prefix?